src/PSR7/SpecFinder.php

```diff
@@ -37,6 +38,39 @@ public function __construct(OpenApi $openApi)
         $this->openApi = $openApi;
     }
 
+    /**
+     * @return Parameter[]
+     *
+     * @throws NoPath
+     */
+    public function findPathSpecs(OperationAddress $addr) : array
```

Comment:
I suggest to tune naming since I was confused for couple of minutes figuring out it's not a recursion method. Maybe something like `findAllPathSpecs` or `findPathAndOperationSpecs`

Reply:
fixed